Example/PrebidDemo/PrebidDemoSwift/InstreamVideoViewController.swift

```diff
@@ -0,0 +1,144 @@
+/*   Copyright 2018-2019 Prebid.org, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import UIKit
+import GoogleInteractiveMediaAds
+import PrebidMobile
+
+class InstreamVideoViewController: UIViewController, IMAAdsLoaderDelegate, IMAAdsManagerDelegate {
+    
+    @IBOutlet var adServerLabel: UILabel!
+    
+    @IBOutlet var appInstreamView: UIView!
+    
+    var adServerName: String = ""
+    
+    private var adUnit: VideoAdUnit!
+    
+    var adsLoader: IMAAdsLoader!
+    var adsManager: IMAAdsManager!
```

Comment:
Please `destroy` adsManager object when it is necessary

Reply:
Please check #337 with Rubicon in-stream

Reply:
done